src/cli/Cli.ts

```diff
@@ -35,16 +37,21 @@ export class Cli {
   public currentCommandName: string | undefined;
   private optionsFromArgs: { options: minimist.ParsedArgs } | undefined;
   private commandsFolder: string = '';
+  private showHelpOnFailure: boolean = false;
   private static instance: Cli;
+  private static conf: any;
 
   private constructor() {
+    Cli.conf = new Configstore(configstore.name);
```

Comment:
Let's change the config to be lazy-loaded so that it's only loaded when it's needed and we can avoid any overhead when running commands that don't need config settings.